.circleci/config.yml

```diff
@@ -68,7 +68,10 @@ jobs:
           tests_directory: ./tests
           <<: *poetry_install
       - store_test_results:
-          path: ".pytest_results"
+          path: '.pytest_results'
+      - run:
```

Comment:
💡 

Maybe cleaner to add a specific step in the deployment pipeline with dedicated JS testing?

Reply:
Yes, added a new CI job for JS testing